blog/_posts/2019-11-13-affiliations-feature.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: blog_post
+title: "Affiliation searches: the Why, What, and How of our Canonical Affiliations Feature"
+author: "Carolyn Stern Grant and Matthew Templeton"
+position: "ADS"
+category: blog
+label: general
+---
+
+One of the features new to ADS since the new interface has launched is search and organization by institutional affiliation.  The 15 million publications in ADS have more than 35 million combined author affiliations.  The ADS have long wanted to have these data in a searchable format, and we introduced this feature in early 2019.  Getting to this point took a lot of manual labor -- in fact, much more than the pipeline that resolves the affiliation strings -- and while this database will never be 100 percent complete, we’re hoping to use machine learning techniques to reduce the amount of human effort required to match affiliation data to their identifiers.
+
+We have said this before but it certainly bears repeating.  While affiliation information is largely complete for recent refereed literature, users need to remember that not every record contains an affiliation, therefore, searching by affiliation alone will inherently be incomplete.  We strongly recommend combining affiliation searches with author searches for best results.
+
+**Using affiliations as part of your search strategy**
+
+Affiliation data for papers are available via the “Affiliations” facet in ADS.  As an example, if you search for a first author, you’ll get a list of papers having that first author, and on the left-hand side of the results page you’ll see ways to refine those results -- who their coauthors are, whether the paper is in the astronomy, physics, or general database, whether it’s a refereed publication or not, and so on.  One of those facets is “Affiliation”, and by clicking it you’ll get a list of institutional affiliations sorted by the number of times that affiliation occurs in a paper with that first author.
```

Comment:
A screenshot of the search results page with the affiliations facet expanded would be great after this paragraph.

Oh, and move the comma inside the quotes on `"Affiliation",` - I think there were a couple of other instances in here with the same issue. On a similar note, is the facet "Affiliation" or "Affiliations"? Make sure this is consistent.

Reply:
facet.png